features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/h2.sql

```diff
@@ -978,6 +978,53 @@ CREATE TABLE IF NOT EXISTS IDN_FED_AUTH_SESSION_MAPPING (
 	PRIMARY KEY(IDP_SESSION_ID)
 );
 
+CREATE TABLE IF NOT EXISTS IDN_CONFIG_TYPE (
+ID          VARCHAR(255)  NOT NULL,
+NAME        VARCHAR(255)  NOT NULL,
+DESCRIPTION VARCHAR(1023) NULL,
+PRIMARY KEY (ID),
+CONSTRAINT TYPE_NAME_CONSTRAINT UNIQUE (NAME)
+);
+
+CREATE TABLE IF NOT EXISTS IDN_CONFIG_RESOURCE (
+ID            VARCHAR(255) NOT NULL,
+TENANT_ID     INT          NOT NULL,
+NAME          VARCHAR(255) NOT NULL,
+CREATED_TIME  TIMESTAMP    NOT NULL,
+LAST_MODIFIED TIMESTAMP    NOT NULL,
+HAS_FILE      BOOLEAN(1)   NOT NULL,
+HAS_ATTRIBUTE BOOLEAN(1)   NOT NULL,
+TYPE_ID       VARCHAR(255) NOT NULL,
+UNIQUE (NAME, ID, TYPE_ID),
```

Comment:
This should be (NAME, TENANT_ID, TYPE_ID) as done in the MySQL script.. check in other db types as well.